src/index.ts

```diff
@@ -140,7 +140,7 @@ export class CapBase<T> {
         "router" in args
           ? args.router
           : await CapRouter.init({
-              host: args.routerHost,
+              host,
```

Comment:
@aleacevedo sorry to bother, but noticed that we don't need a separate host field name and the default field host should be used. This is because there's no use-case where the CapRouter and CapRoot hosts would differ or the developer doesn't need so much control, as that'd be a bit of an inconvenience. I'm pointing this out to you, because you might be using the hostRouter somewhere in your code (did a quick check in our repositories and that is not the case, but if there is you'll have to update at some point, unless you vote against this change). Let me know! Thanks

https://github.com/search?q=org%3APsychedelic+hostRouter&type=code